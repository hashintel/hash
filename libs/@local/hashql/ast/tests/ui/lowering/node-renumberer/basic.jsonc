//@ run: pass
//@ description: ensure renumbering works correctly
["+", { "#literal": 3 }, { "#literal": 4 }]
