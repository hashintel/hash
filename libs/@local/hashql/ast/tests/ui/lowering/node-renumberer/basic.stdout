Expr#0@8
  ExprKind (Call)
    CallExpr#1@8
      Expr#2@3
        ExprKind (Path)
          Path#3@3 (rooted: true)
            PathSegment#4@2 (name: math)
            PathSegment#5@2 (name: add)
      Argument#6@5
        Expr#7@5
          ExprKind (Literal)
            LiteralExpr#8@4
              LiteralKind (Integer)
                IntegerLiteral (3)
      Argument#9@7
        Expr#10@7
          ExprKind (Literal)
            LiteralExpr#11@6
              LiteralKind (Integer)
                IntegerLiteral (4)
